docs/user-guide/rules/validate-set-cookie-header.md

```diff
@@ -0,0 +1,179 @@
+# Validate `Set-Cookie` Header (`validate-set-cookie-header`)
+
+This rule validates the `set-cookie` header and confirms that the
+`Secure` and `HttpOnly` directives are defined when sent from a
+secure origin (HTTPS).
```

Comment:
I think this comment is obsolete as we are now validating other directives. I'm ok with `validate-set-cookie-header`.